### PR TITLE
chore(examples-web): bump web examples dependencies and use exporter-trace-otlp-http

### PR DESCRIPTION
**Pull request by tsloughter**
## Which problem is this PR solving?

- The web examples used the deprecated exporter package `exporter-otlp-http`

## Short description of the changes

- Switched the use of `exporter-otlp-http` to `exporter-traces-otlp-http` in the web examples and updated the other otel dependencies.

### CODE REVIEW

File: examples/web/package.json [hunk 1]
```diff
@@ -27,28 +27,25 @@
...
-    "@opentelemetry/api": "^1.0.3",
-    "@opentelemetry/auto-instrumentations-web": "~0.26.0",
-    "@opentelemetry/context-zone": "~1.0.0",
-    "@opentelemetry/core": "~1.0.0",
-    "@opentelemetry/exporter-otlp-http": "~0.26.0",
+    "@opentelemetry/api": "^1.4.1",
+    "@opentelemetry/auto-instrumentations-web": "^0.32.2",
+    "@opentelemetry/context-zone": "^1.13.0",
+    "@opentelemetry/exporter-trace-otlp-http": "^0.39.1",
     "@opentelemetry/instrumentation": "~0.26.0",
```
Comment (Flarna) [inline examples/web/package.json:43]: why update some deps but not others? This likely results in duplicates in node_modules folder

Comment (tsloughter) [reply examples/web/package.json:43]: I just missed some. Will update.

Comment (tsloughter) [reply examples/web/package.json]: Got them all now.

File: examples/web/package.json [hunk 2]
```diff
@@ -27,29 +27,26 @@
...
-    "@opentelemetry/propagator-b3": "~1.0.0",
-    "@opentelemetry/resources": "~1.0.0",
-    "@opentelemetry/sdk-trace-base": "~1.0.0",
-    "@opentelemetry/sdk-trace-web": "~1.0.0",
-    "@opentelemetry/semantic-conventions": "~1.0.0"
+    "@opentelemetry/api": "^1.4.1",
+    "@opentelemetry/auto-instrumentations-web": "^0.32.2",
+    "@opentelemetry/context-zone": "^1.13.0",
+    "@opentelemetry/exporter-trace-otlp-http": "^0.39.1",
+    "@opentelemetry/instrumentation": "~0.39.1",
```
Comment (Flarna) [inline examples/web/package.json:43]:
```suggestion
    "@opentelemetry/instrumentation": "^0.39.1",
```
to stay consistent with the line above

File: examples/web/package.json [hunk 3]
```diff
@@ -27,29 +27,26 @@
...
+    "@opentelemetry/instrumentation-document-load": "^0.32.2",
+    "@opentelemetry/instrumentation-user-interaction": "^0.32.3",
+    "@opentelemetry/instrumentation-xml-http-request": "^0.39.1",
+    "@opentelemetry/propagator-b3": "^1.13.0",
+    "@opentelemetry/sdk-trace-web": "^1.13.0",
+    "@opentelemetry/semantic-conventions": "~1.13.0"
```
Comment (Flarna) [inline examples/web/package.json:49]:
```suggestion
    "@opentelemetry/semantic-conventions": "^1.13.0"
```
to stay consistent with the line above

**Review by Flarna — APPROVED** [review]

**Pull request merged**